custom-routes-vis/src/main/webapp/script.js

```diff
@@ -99,49 +98,6 @@ function updateCoordinates(lat, lng, containerId) {
       "Latitude: " + lat.toFixed(6) + "<br>Longitude: " + lng.toFixed(6) + "<br>";
 }
 
-/**
```

Comment:
Is this just not needed anymore?

Reply:
Yes I'm not sure why you removed all of Natalie's code here

Reply:
Ah I see you moved it below.

Reply:
Yeah I moved it because it's only called from selectRoute